runtime/doc/lsp.txt

```diff
@@ -862,221 +1018,301 @@ workspace_symbol({query})                     *vim.lsp.buf.workspace_symbol()*
 
 
 ==============================================================================
-Lua module: vim.lsp.log                                              *lsp-log*
+Lua module: vim.lsp.diagnostic                                *lsp-diagnostic*
 
-get_filename()                                    *vim.lsp.log.get_filename()*
-                Returns the log filename.
+                        *vim.lsp.diagnostic.buf_clear_displayed_diagnostics()*
+buf_clear_displayed_diagnostics({bufnr}, {client_id},
+                                {diagnostic_ns}, {sign_ns})
+                Clears the currently displayed diagnostics
 
-                Return: ~
-                    (string) log filename
+                Parameters: ~
+                    {bufnr}          number The buffer number
+                    {client_id}      number the client id
+                    {diagnostic_ns}  number|nil Associated diagnostic
+                                     namespace
+                    {sign_ns}        number|nil Associated sign namespace
 
-set_level({level})                                   *vim.lsp.log.set_level()*
-                Sets the current log level.
+                                         *vim.lsp.diagnostic.buf_get_counts()*
+buf_get_counts({bufnr}, {client_id}, {kind})
```

Comment:
Here you mention `buf_get_counts` but on line 1045 it says `buf_diagnostics_count`

Should it be `buf_get_diagnostics_count` since there is also `buf_clear_displayed_diagnostics` or should `diagnostics` be left out as the scope is known already?